inc/abstracttarget.class.php

```diff
@@ -1634,6 +1759,24 @@ public function prepareInputForUpdate($input) {
          $input['uuid'] = plugin_formcreator_getUuid();
       }
 
+      switch ($input['commonitil_validation_rule']) {
```

Comment:
There is a failure in unit tests here : This code makes commonitil_validation_rule mandatory. As some users begin to use the plugin via API, it would be better to check if commonitil_validation_rule and commonitil_validation_question are set before processing them. This allows an API client to not mention them if the defaults are OK.